packages/uploadthing/src/sdk/index.ts

```diff
@@ -278,3 +278,36 @@ export const renameFile = async (updates: Rename | Rename[]) => {
 
   return json;
 };
+
+export const getUsageInfo = async () => {
+  guardServerOnly();
```

Comment:
Does this actually need to be server-only?

Reply:
we need the api key so yea?

Reply:
🤦 You're right, I am dumb....